client/network.go

```diff
@@ -53,7 +53,7 @@ func (cli *NetworkCli) CmdNetworkCreate(chain string, args ...string) error {
 
 	nc := networkCreate{Name: cmd.Arg(0), NetworkType: *flDriver}
 
-	obj, _, err := readBody(cli.call("POST", "/networks/name/"+cmd.Arg(0), nc, nil))
+	obj, _, err := readBody(cli.call("POST", "/networks", nc, nil))
```

Comment:
I understand that this change is required in client. But this is not the only change. There are 4 functions with now deprecated API usage. (am adding more to it now).
I prefer if we can remove this change from client and handle it all together in a client-only PR.

(also the Tests didnt catch such api changes as it looks only for hard Errors and not soft HTTP header based errors)